docs/features/techdocs/using-cloud-storage.md

```diff
@@ -37,15 +37,17 @@ techdocs-backend will publish documentation to this bucket. TechDocs will fetch
 files from here to serve documentation in Backstage. Note that the bucket names
 are globally unique.
 
-Set the config `techdocs.publisher.googleGcs.bucketName` in your
-`app-config.yaml` to the name of the bucket you just created.
+Set the config `techdocs.publisher.googleGcs.bucketName` and
+`techdocs.publisher.googleGcs.bucketName` in your `app-config.yaml` to the name
```

Comment:
Maybe `techdocs.publisher.googleGcs.projectId` needs a separate comment ?

Reply:
Good point - I've rephrased this.